agent/bench-scripts/tests/pbench-trafficgen/test-40.txt

```diff
@@ -103,4 +103,5 @@ Iteration 3-5000000-revunidirectional-74B-1000flows-0.0pct_drop complete (3 of 3
 /var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-postprocess-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/3-5000000-revunidirectional-74B-1000flows-0.0pct_drop/sample1
 /var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-tool-meister-start --sysinfo=none default
 /var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-tool-meister-stop --sysinfo=none default
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/require-rpm dpdk-tools  
```

Comment:
Because trafficgen doesn't provide the `match` argument to `check_install_rpm`.

Reply:
Can you expound on the implications of this?  (E.g., is this supposed to mean that pbench will work with _any_ version of trafficgen?  Or, does it mean that will it work with only a specific version and will silently break if that version isn't present?  Or,...?)

Reply:
Well this is really a larger problem where we have placed into `pbench-trafficgen` the requirement that the RPM `dpdk-tools` is installed.  That should be a `trafficgen` concern.  We should only be concerned that `trafficgen` is installed.

Aside from that, it should mean that we'll work with *any* version of `dpdk-tools`, which means the code in this PR is broken because we default to `eq`.  We need a notion of `any` to support when a version is not given.

Reply:
> We need a notion of `any` to support when a version is not given.

I would think that "any" could be expressed by something like `0.0.0 ge`.

And, I concur that "any" would be the appropriate default when no version is given.  If a version is given without any qualification, then having the default for the "match" be `eq` seems reasonable.  (And, when the version is not specified, I would either error-out if the match is specified or just ignore it if it is not `ge`.)

Reply:
We can't use `0.0.0` because there are different formats for versions, so I have added the notion of `any` to just find the rpm at all.